tests/suites/test_suite_asn1parse.data

```diff
@@ -608,25 +608,33 @@ Find named data: first match
 find_named_data:"414141":"414141":"434343":"444444":"414141":0:0
 
 Free named data: null pointer
+depends_on:MBEDTLS_TEST_DEPRECATED
 free_named_data_null:
 
 Free named data: all null
+depends_on:MBEDTLS_TEST_DEPRECATED
 free_named_data:0:0:0
 
 Free named data: with oid
+depends_on:MBEDTLS_TEST_DEPRECATED
 free_named_data:1:0:0
 
 Free named data: with val
+depends_on:MBEDTLS_TEST_DEPRECATED
 free_named_data:0:1:0
 
 Free named data: with next
+depends_on:MBEDTLS_TEST_DEPRECATED
 free_named_data:0:0:1
 
 Free named data list (empty)
+depends_on:MBEDTLS_TEST_DEPRECATED
```

Comment:
`free_named_data_list` don't have to depend on `MBEDTLS_TEST_DEPRECATED`.

Reply:
@AndrzejKurek I do not understand.  The tests I added deprecation to call the test functions `free_named_data_null()` or `free_named_data()` and both of those test functions call the deprecated function `mbedtls_asn1_free_named_data()`.

Reply:
You also added `depends_on:MBEDTLS_TEST_DEPRECATED` to `free_named_data_list` tests which only call `mbedtls_asn1_free_named_data_list`. This one is not deprecated and doesn't call any other deprecated functions.

Reply:
Ah.  I see.  I removed the `MBEDTLS_TEST_DEPRECATED` tag from the tests which use `free_named_data_list`
:rocket: